plasmapy/plasma/grids.py

```diff
@@ -557,23 +557,6 @@ def dax2(self):
         """
         return self._get_dax(axis=2)
 
-    @property
-    def grid_resolution(self):
-        r"""
-        A scalar estimate of the grid resolution.
-
-        For uniform grids, this is the minima of [dax0, dax1, dax2].
-
-        For non-uniform grids, it is the closest spacing between any two points.
-        """
-
-        if self.is_uniform:
-            return min(self.dax0, self.dax1, self.dax2)
-        else:
-            distances = distance.cdist(self.grid, self.grid)
-            np.fill_diagonal(distances, np.inf)
-            return np.min(distances)
-
```

Comment:
Seems like this, given it's implemented in the subclasses, could also be an abstract method ([like this](https://docs.python.org/3/library/abc.html#abc.abstractproperty)).

Reply:
Yeah, that's tougher. In the cylindrical case I'd expect this to mean "smallest of the three grid resolutions, provided the angular resolution is..."  Oh. Yeah, because the angular resolution aka arc length is going to be a function of distance from the Z axis, this would also be a function of the position... Yeah, let's skip this one. Or we could just decide to do min(Res_r, Res_z). But that seems arbitrary until we encounter a use case.